doc/doxygen/src/ForManagers.dox

```diff
@@ -197,7 +197,7 @@
  * In order to support entity relationships:
  *
  * - Implement the API methods grouped under
- *   @ref openassetio.managerApi.ManagerInterface.ManagerInterface
+ *   @fqref{managerApi.ManagerInterface}.ManagerInterface
```

Comment:
[fix] This links to the `ManagerInterface` constructor, probably not what was intended.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1058/commits/948efe5518f4da207ec3438f17531c456b524965